start.php

```diff
@@ -31,11 +31,36 @@
 
 	global $fs_active_plugins;
 
-	$this_sdk_relative_path = plugin_basename( dirname( __FILE__ ) );
+    $paths = array(
+        'file_path'          => __FILE__,
+        'current_theme_path' => trailingslashit( get_stylesheet_directory() ),
+        'themes_directory'   => trailingslashit( get_theme_root() )
+    );
+
+    if ( function_exists( 'wp_normalize_path' ) ) {

```

Comment:
@fajardoleo why not to put the normalization code below the `fs-essential-functions.php` include with `fs_normalize_path()`?